src/main/java/com/google/step/YTLounge/servlets/RetrieveQueue.java

```diff
@@ -0,0 +1,113 @@
+package com.google.step.YTLounge.servlets;
```

Comment:
Please consider adding tests for these changes.

Reply:
I will do that in a new PR